inscriptions/ISic003637.xml

```diff
@@ -185,14 +190,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/><gap reason="illegible" quantity="1" unit="character"/> <name><supplied reason="lost">Cl</supplied>audius Iu<supplied reason="lost">st</supplied>us</name> I<supplied reason="lost">Ivir et patro</supplied> 
```

Comment:
I think this should be done following the version in EDR?
[http://www.edr-edr.it/edr_programmi/res_complex_comune.php?do=book&id_nr=EDR112554](http://www.edr-edr.it/edr_programmi/res_complex_comune.php?do=book&id_nr=EDR112554)